site/docs/skylark/backward-compatibility.md

```diff
@@ -662,5 +663,29 @@ my_rule = rule(
 *   Default: `false`
 *   Introduced in: `0.18.0`
 
+### Disallow `cfg = "data"`
+
+`cfg = "data"` is a no-op that incorrectly gives the impression dependencies under
+it are built in some special "data" mode:
```

Comment:
Nit: "some special 'data' mode" -- maybe say it's an old mode that's obsolete, instead of pretending it never existed :)

Reply:
Done.